### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards> InputFormWithLabels.json> Phone Number]: Screen reader does not announce the information of 'Maximum character limit reached' on the 'Phone number' edit field.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8191]
### Target Platforms

UWP

### SDK Version

3.0.2210.26001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25267.1000)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2210.26001
Screen reader: Narrator

**Repro Steps:**

1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ Windows+ Enter key.
3. Navigate to the 'InputFormWithLabels.json' through tab key and activate it.
4. Now, navigate to 'Phone Number' through tab key.
5. Observe the issue.

**Actual Result:**
Screen reader does not notify about the "maximum character limit reached " to the user when user provide input in the 'phone number' edit field.

**Expected Result:**
Screen reader should notify about the "**maximum character limit reached** " to the user when user provide input in the 'phone number' edit field.

**Notes:**
Validation is not implemented on 'Phone Number' edit field so that user can't be able to type alphabet  in the same.

**User Impact:**
Screen reader user will not be able to track or will continue to enter the data in the edit field if maximum limit reached information is not announced.

**MAS Reference link:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EXdDUZzldv5MgwDaMHnTzcMBcT8opKGncYjU-gCJ4uqQdg?e=9nuVmt

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/104483005/209768239-cc109e12-45a1-4c17-9f6b-73c41d1176bc.mp4

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Note: This a11y bug is a feature request for dynamic input validation

**Comment by vagpt** [comment]
This issue is working fine, hence closing this issue.

![image](https://github.com/user-attachments/assets/4216395e-2850-4556-96de-dd2cd2862415)
